browser-interface/packages/shared/world/SceneWorker.ts

```diff
@@ -329,7 +329,7 @@ export class SceneWorker {
 
     sceneEvents.emit(SCENE_LOAD, signalSceneLoad(this.loadableScene))
 
-    const WORKER_TIMEOUT = 90_000 // ninety seconds to mars
+    const WORKER_TIMEOUT = 120_000
```

Comment:
Why is this needed?

Reply:
Further information [here](https://decentralandteam.slack.com/archives/C0292P4HJ93/p1678985200846279)

Reply:
@menduz I've heard from back channels that this change is needed for MFW and there is not enough time for a proper fix

Reply:
Never change parameters like that without adding a comment explaining why it was changed, pleeease 🙏

Nothing prevents anyone to revert it and it only makes the codebase more fragile

// timeout is 120 because gltf are big and network is slow

Even that example would be helpful... Right now nobody knows why it was changed, what it fixes and what is the root cause of the problem.